### PR TITLE
test: setup playwright e2e tests

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: tests/tenant.test.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {test, expect} from '@playwright/test';
+
+test.describe('Test Tenants page', async () => {
+    test('Tenants page is OK', async ({page}) => {
+        const response = await page.goto('/tenants');
```
Comment (sareyu) [inline tests/tenant.test.ts:5]: good for start, later pls take a look at Page Object Model approach - afaik it's recommended by playwright maintainers

**Review by sareyu — APPROVED** [review]

File: tests/README.md
Comment (ariser) [inline tests/README.md:1]: I'd put this docs in the main readme, this is a part of the development process

Comment (artemmufazalov) [reply tests/README.md:1]: I agree with you, it is hard to notice it there. I'll move it to main README

File: README.md [hunk 2]
```diff
@@ -4,27 +4,50 @@ Local viewer for YDB clusters
 
 ## How to work with this repo
 
-### Development
+## Development
```
Comment (ariser) [inline README.md:7]: you now have nested headings of the same level

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -34,3 +34,31 @@ jobs:
...
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Install Plawright deps
+        run: npm run test:e2e:install
+
+      - name: Run Playwright tests
+        run: npm run test:e2e
```
Comment (zeruk) [inline .github/workflows/ci.yml:64]: You can extract test results as artefacts if you want

**Pull request merged**